lms/djangoapps/learner_recommendations/utils.py

```diff
@@ -55,6 +55,67 @@ def _remove_user_enrolled_course_keys(user, course_keys):
     return enrollable_course_keys
 
 
+def _has_country_restrictions(product, user_country):
+    """
+    Helper method that tell whether courses or programs have any country restrictions.
+    Args:
+      product: course/program
+      user_country (string): country the user is logged in from
+
+    Returns:
+        True if the product is restricted in the country and False otherwise
+    """
+    if not user_country:
+        return False
+
+    allow_list, block_list = [], []
+    location_restriction = product.get("location_restriction", None)
+    if location_restriction:
+        restriction_type = location_restriction.get("restriction_type")
+        countries = location_restriction.get("countries")
+        if restriction_type == "allowlist":
+            allow_list = countries
+        if restriction_type == "blocklist":
+            block_list = countries
+
+    return user_country in block_list or (allow_list and user_country not in allow_list)
+
+
+def _parse_course_owner_data(owner):
+    """
+    Helper to parse course owner data.
+    """
+    return {
+        "key": owner.get("key"),
+        "name": owner.get("name"),
+        "logo_image_url": owner.get("logo_image_url")
+    }
+
+
+def course_data_for_course_about_page_recommendations(course_data):
```

Comment:
Lets move this as a private function like we did for dashboard APIs.

Reply:
I think utils is the best place for helper functions. What do you think about that?

Reply:
The other option is to rename it
```suggestion
def _course_data_for_discovery_card(course_data):
```